fetchcode/package.py

```diff
@@ -327,3 +328,26 @@ def get_rubygems_data_from_purl(purl):
         download_url=download_url,
         **purl.to_dict(),
     )
+
+
+@router.route("pkg:brew/.*")
+def get_homebrew_data_from_purl(purl):
```

Comment:
@pombredanne , should it be homebrew or brew ?

Reply:
homebrew IMHO is the name

Reply:
@nk4456542 please do the changes :)

Reply:
yeah sure.

Reply:
@TG1999  made the change :)